immuni_analytics/core/config.py

```diff
@@ -67,16 +66,14 @@
 EXPOSURE_PAYLOAD_QUEUE_KEY: str = config(
     "EXPOSURE_PAYLOAD_QUEUE_KEY", default="ingested_exposure_data"
 )
-
+PROXY_URL: Optional[str] = config("PROXY_URL", default=None)
```

Comment:
DEVICE_CHECK_PROXY_URL?

Reply:
Is it tied to the device check? In our implementation, yes, but potentially the request.py can be called from anywhere.

Reply:
uh ok, if it is a generic helper then 👍